### ISSUE TITLE
verbose flag for `bitmagnet process`

### ISSUE THREAD

**Issue by leofidus** [bitmagnet-io/bitmagnet#309]
- [x] I have checked the existing issues to avoid duplicates
- [x] I have redacted any info hashes and content metadata from any logs or screenshots attached to this issue

### Is your feature request related to a problem? Please describe

I'm writing a classifier config for my bitmagnet instance. This involves a loop of modifying the classifier.yml file, testing the changes with `bitmagnet process --infoHash=deadbeefredacted` and checking the results in the web interface. The last step is quite cumbersome, and I'm often left wondering if my config doesn't work or I'm seeing caching effects

### Describe the solution you'd like

An additional flag for `bitmagnet process` that outputs for each processed file which actions were taken (delete, add-tag, content-type)

### Describe alternatives you've considered

Running with LOG_LEVEL=debug or directly looking at the database both work as alternative solutions. But the former is very verbose (and requires reading SQL statements), and the latter is requires connecting to the database.

Instead of just outputting results, the new flag could also output other useful information. When debugging classifier configs it would sometimes be useful to get step-by-step output what the result of conditions or branches was. This might go a a bit far though

**Comment by mgdigital** [comment]
I agree something better tracing abilities would be useful. An issue I can see is the full trace of the paths followed might be difficult to interpret (and to show clearly) so would need some thought. So leaving this open for now. In the meantime adding tags are a way you can currently use to see if any particular branch was reached...
